moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1208,7 +1214,8 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     if (constraints_storage_)
     {
       moveit_warehouse::ConstraintsWithMetadata msg_m;
-      if (constraints_storage_->getConstraints(msg_m, constraint, robot_model_->getName(), opt_.group_name_))
+      bool bb = constraints_storage_->getConstraints(msg_m, constraint, robot_model_->getName(), opt_.group_name_);
+      if ((bb != false) && msg_m)
```

Comment:
Is the checking method of msg_m appropriate?
Even bool type describes ```! = false```.

Reply:
Appropriate because it matches the format of the pointer variable comparison on line 1208. (msg_m is of type shared_ptr)

Reply:
OK.